test/app_po.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:async';
+
+import 'package:ngpageloader/pageloader.dart';
+
+part 'app_po.g.dart';
+
+@PageObject()
+abstract class AppPO {
+  AppPO();
+  factory AppPO.create(PageLoaderElement context) = $AppPO.create;
+
+  @ByTagName('h1')
+  PageLoaderElement get _title;
+
+  String get title => _title.visibleText;
+
+  @ByTagName('li')
+  List<PageLoaderElement> get _heroes;
+
+  Iterable<Map> get heroes =>
+      _heroes.map((el) => _heroDataFromLi(el.visibleText));
+
+  // ···
```

Comment:
what's this comment for?

Reply:
The comment line was copied from the Tutorial by mistake.  I will remove it.